Makefile

```diff
@@ -854,6 +854,9 @@ mitre:
 .PHONY: bootstrap_migration
 bootstrap_migration:
 	$(SILENT)if [[ "x${DESCRIPTION}" == "x" ]]; then echo "Please set a description for your migration in the DESCRIPTION environment variable"; else go run tools/generate-helpers/bootstrap-migration/main.go --root . --description "${DESCRIPTION}" ;fi
+.PHONY: fix_migrations
+fix_migrations:
+	$(SILENT)if [[ "x${MIGRATION_PACKAGES}" == "x" ]]; then echo "Please set the MIGRATION_PACKAGES environment variable"; echo "MIGRATION_PACKAGES can be a comma separated list of migration packages to fix"; else ./scripts/fix-migrations.sh; fi
```

Comment:
The `package` here is the name of the migration directory and not the go package name right?
So `m_213_to_m_214_testing_bootstrap_migration` and not `m213tom214`